config/config.toml

```diff
@@ -25,11 +25,24 @@ enable_allow_all_cors = true
 # Storage Configuration
 [services]
 service_endpoint = "http://localhost:3000"
-storage = "redis"
 
-[services.storage_option]
-address = "localhost:6379"
-password = ""
+# example bolt config with filepath option
+storage = "bolt"
+
+[[services.storage_option]]
+id = "boltdb-filepath-option"
+option = "bolt.db"
```

Comment:
I'm wary of running with bolt, since it might hide some issues of the default way in which we are intending to run the service.

Reply:
this shouldn't matter since compose/CI run with redis

Reply:
this shouldn't matter since compose/CI run with redis

Reply:
I think it matters because we're having significant differences between running así service locally, and what's being run in CI. I think we should strive to have the differences be minimal, so that it's easy to locally reproduce problems that happen in CI.

Reply:
there are reasons to just run the service in isolation, and to run it in a prod-like environment.

the prod-like environment is using compose. bolt is suitable for running the service in isolation.

we've also taken the approach of introducing a storage abstraction. if you're asserting that our abstraction is faulty then we should fix that -- either by making it better or removing it entirely.